vembrane/representations.py

```diff
@@ -101,6 +101,11 @@ def __init__(
         self._info_dict = {}
 
     def __getitem__(self, item):
+        if item == "END":
+            # pysam removes END from info. In order to fit with user expectations,
+            # (they will expect INFO["END"] to work) we emulate it being present by
+            # inferring it from pysams record representation.
+            return get_end(self._record)
```

Comment:
I think this should be in the `except` arm, and the `_info_dict` should be updated accordingly, just like every other access.

Reply:
Good point.